### PR TITLE
Change sdg-image-alt text in the lucky-parking.md

### LINKED ISSUE #3097: Change alt text for sdg image in lucky parking md file
### Overview
We need to ensure that alt text for images adhere to Web Content Accessibility Guidelines (WCAG). For this issue, we will change the alt text for the lucky parking project's sustainability development goal image on [citizen engagement page](https://www.hackforla.org/citizen-engagement).

### Details
Designers and PMs identified that some alt texts in the Citizen Engagement page should be changed because they are either inaccurate or too long. Ideally, alt text should be descriptive but short and succinct. According to the WCAG page, an accompanying alt text should be short. Additionally, the Functional Accessibility Evaluator (FAE) recommends for alt text to be no more than 100 characters.

### Action Items
- [x] For the file `_projects/luck-parking.md`, change [line 62](https://github.com/hackforla/website/blob/f2ce8983e77a9b80d455e3677b1cf323092c4f8d/_projects/lucky-parking.md?plain=1#L62) for the alt text:

from
```
sdg-image-alt: sustainable cities bottom sustainable dev goal
```
to
```
sdg-image-alt: '11: sustainable cities and communities'
```
- [x] **For merge team**: Once the pull request associated with this issue is approved and merged, please update and edit epic #3089 by
  - [x] Checking off the dependency for this issue
  - [x] If all dependencies are checked off, please move issue #3089 to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- This issue is part of epic #3089 
- [Citizen Engagement page](https://www.hackforla.org/citizen-engagement)
- [Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)

#### File and code links you will need to work on this issue
- Directory to find the page once you have it in your IDE: `_projects/lucky-parking.md`
- Click on the link below to see the source file code:
https://github.com/hackforla/website/blob/f2ce8983e77a9b80d455e3677b1cf323092c4f8d/_projects/lucky-parking.md?plain=1#L62

#### Information about WCAG
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
- [WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)

### PR DESCRIPTION
**Pull request by plang-psm**
Fixes #3097

### What changes did you make and why did you make them ?

  - change the sdg-image-alt text in the lucky-parking.md file to '11: sustainable cities and communities' from sdg-image-alt: sustainable cities bottom sustainable dev goal

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

No visual changes were made.

### CODE REVIEW

**Comment by neevliberman** [comment]
Review ETA: 3 PM PST, 9-5-22
Availability: 3 Hours

File: _projects/lucky-parking.md [hunk 1]
```diff
@@ -55,9 +55,9 @@ status: Active
...
 impact: Our project seeks to educate and inform city leaders and the community about the effects of Los Angeles’ parking policies, hopefully serving as a tool in discussing more equitable solutions to our transportation problems.
-sdg: "<strong>11.2:</strong> By 2030, provide access to safe, affordable, accessible and sustainable transport systems for all, improving road safety, notably by expanding public transport, with special attention to the needs of those in vulnerable situations, women, children, persons with disabilities and older persons."
+sdg: '<strong>11.2:</strong> By 2030, provide access to safe, affordable, accessible and sustainable transport systems for all, improving road safety, notably by expanding public transport, with special attention to the needs of those in vulnerable situations, women, children, persons with disabilities and older persons.'
 card-image-src: /assets/images/projects/lucky-parking.png
-card-image-alt: 
+card-image-alt:
```
Comment (neevliberman) [inline _projects/lucky-parking.md:60]: Why was this change made? Does the extra white space cause issues?

**Review by neevliberman — DISMISSED** [review]
I see that you changed all double quotes to single quotes. Was this for consistency or is there also a functional purpose? Also on line 60 you deleted a whitespace character, I'm also curious about this. Doesn't seem website-breaking but just in case it's good to know why certain changes were made, so it might be helpful to explain in your comment! The change you needed to make was good, otherwise!

**Review by arpitapandya — APPROVED** [review]
The changes are correct! Because it doesn't break anything I would approve this. But, I would suggest not to make extra changes for the issues which are about to make line specific change only. Sometimes, that may break other things. Here the issue suggest to make changes to only line 60. Great job @plang-psm

**Comment by plang-psm** [comment]
Thank you for catching this! @neevliberman! I only edited line 60 but I believe my editor may have auto formatted once I saved the line with single quotes. I usually use single quotes when coding so thats most likely it. I'll take a look into it to make sure it doesn't happen in the future.

**Comment by plang-psm** [comment]
Thank you @arpitapandya! I believe the extra changes came from my editor as it may have auto formatted when I saved the file. I'll make sure to change this to prevent any future unwanted changes.

**Comment by arpitapandya** [comment]
> Thank you @arpitapandya! I believe the extra changes came from my editor as it may have auto formatted when I saved the file. I'll make sure to change this to prevent any future unwanted changes.

@plang-psm Since, the changes has already been merge and updated now, you do not do anything. No problem

**Pull request merged**